devel/fortran-utils/Portfile

```
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+
+name            fortran-utils
+version         1.2
+revision        0
+categories      devel
+platforms       darwin
+license         BSD
+maintainers     {@kamischi web.de:karl-michael.schindler} openmaintainer
+homepage        https://sourceforge.net/projects/fortran-utils/
+master_sites    sourceforge
+extract.suffix  .tar.bz2
+worksrcdir      {}
+checksums       rmd160  353e753b2084f3eab5cf9cfd9c2ccf4e983f05b5 \
+                sha256  b2a25ae867a7755fb8d839ca5dd59a24888d12d217aea6ddfd57aceda888b197 \
+                size    7689
+
+description     Tools for use with Fortran77 code
+
+long_description \
+    The fortran-utils are: \
+    \n- fpr: transform Fortran's carriage control conventions to UNIX line printer conventions \
+    \n- fsplit: split a multi-routine Fortran file into individual files
+
+notes "This version has patches from FreeBSD and NetBSD."
+
+supported_archs noarch
+installs_libs   no
+use_configure   no
+build {
+    system -W ${worksrcpath}/fsplit "${configure.cc} -o fsplit fsplit.c"
+    system -W ${worksrcpath}/fpr    "${configure.cc} -o fpr    fpr.c"
```

Now you're using the right compiler, but you're still not using the right compiler or linker flags (including `-arch` flags) nor offering a universal variant. This does part of it:
```suggestion
    foreach p [list fpr fsplit] {
        system -W ${worksrcpath}/${p} "${configure.cc} ${configure.cflags} ${configure.ldflags} [get_canonical_archflags cc] -o ${p} ${p}.c"
    }
```
But this project includes a Makefile. We should be using that rather than doing the building and installing manually. The makefile portgroup can help.

---

I can't recall the exact details when I tried to use the Makefile initially. But after a few tries to tweak it, I found it easier to simply call the compiler and linker and install the results. If you find this unacceptable, I can try again.

---

The azure pipeline failed after this commit. If I understand the log right, install failed, but I do not get it, why.

---

I have a version of the portfile that works with the existing makefiles which I'd like to commit to this PR if that's alright with you.